pod/video/models.py

```diff
@@ -26,7 +26,11 @@
 from ckeditor.fields import RichTextField
 from tagging.fields import TagField
 
+import select2.fields
+import select2.models
```

Comment:
remove select2.models, it's unused. Can you replace import select2.fields by from select2 import fields as select2_fields and change it inside the file